apps/discord-bot/src/commands/leaderboards/player-leaderboard.command.ts

```diff
@@ -37,17 +37,37 @@ import {
   WARLORDS_MODES,
   WOOLWARS_MODES,
 } from "@statsify/schemas";
-import { ApiService, Command, CommandContext, SubCommand } from "@statsify/discord";
+import {
+  ApiService,
+  ChoiceArgument,
+  Command,
+  CommandContext,
+  SubCommand,
+} from "@statsify/discord";
 import { BaseLeaderboardCommand } from "./base.leaderboard-command";
 import { GamesWithBackgrounds, mapBackground } from "#constants";
 import { GuildLeaderboardArgument } from "./guild-leaderboard.argument";
 import { GuildLeaderboardSubCommand } from "./guild-leaderboard.subcommand";
-import { PlayerLeaderboardArgument } from "./player-leaderboard.argument";
+import {
+  PlayerLeaderboardArgument,
+  SHORT_TO_LONG_HISTORICAL_TYPE,
+} from "./player-leaderboard.argument";
 import { getBackground } from "@statsify/assets";
 
+const HISTORICAL_ARGUMENT = new ChoiceArgument({
```

Comment:
Have we decided whether or not this should be a required option or not?

Reply:
not yet